packages/network-controller/src/NetworkController.ts

```diff
@@ -63,6 +65,14 @@ const name = 'NetworkController';
 
 export type EthQuery = any;
 
+type Provider = any;
```

Comment:
In order to not make this `any` we'd have to backfill a lot of the types for `web3-provider-engine`. Given that we're about to replace that package, I decided to defer using a real type here until we've made that replacement.